version.props

```diff
@@ -38,12 +38,10 @@
 
     <!-- Run the build with /p:IsFinalBuild=true to produce the product with 'final' branding and versioning -->
     <IsFinalBuild Condition=" '$(IsFinalBuild)' == '' ">false</IsFinalBuild>
-    <IncludePreReleaseLabelInPackageVersion>true</IncludePreReleaseLabelInPackageVersion>
-    <IncludePreReleaseLabelInPackageVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(PreReleaseLabel)' == 'servicing' OR '$(PreReleaseLabel)' == 'rtm')">false</IncludePreReleaseLabelInPackageVersion>
+    <IncludePreReleaseLabelInVersion>true</IncludePreReleaseLabelInVersion>
+    <IncludePreReleaseLabelInVersion Condition=" '$(IsFinalBuild)' == 'true' AND ('$(PreReleaseLabel)' == 'servicing' OR '$(PreReleaseLabel)' == 'rtm')">false</IncludePreReleaseLabelInVersion>
 
-    <!-- The version in files -->
-    <PackageVersion>$(VersionPrefix)</PackageVersion>
-    <PackageVersion Condition=" '$(IncludePreReleaseLabelInPackageVersion)' == 'true' ">$(VersionPrefix)-$(VersionSuffix)</PackageVersion>
+    <VersionSuffix Condition=" '$(IncludePreReleaseLabelInVersion)' == 'false' "></VersionSuffix>
```

Comment:
I think a simpler fix would be to leave the file as is and just add 
```xml
<Version>$(PackageVersion)</Version>
```
VersionPrefix/VersionSuffix end up in other places, like informational version attributes, which is useful for figuring out the exact build number used to produce our packages and binaries.